packages/rfw/example/wasm/lib/main.dart

```diff
@@ -0,0 +1,94 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// This file is hand-formatted.
+
+import 'dart:async';
+import 'dart:io';
+
+import 'package:flutter/rendering.dart';
+import 'package:flutter/widgets.dart';
+import 'package:path/path.dart' as path;
+import 'package:path_provider/path_provider.dart';
+import 'package:rfw/rfw.dart';
+import 'package:wasm/wasm.dart';
+
+const String urlPrefix = 'https://raw.githubusercontent.com/flutter/packages/master/packages/rfw/example/wasm/logic';
+
+const String interfaceUrl = '$urlPrefix/calculator.rfw';
+const String logicUrl = '$urlPrefix/calculator.wasm';
+
+void main() {
+  runApp(WidgetsApp(color: const Color(0xFF000000), builder: (BuildContext context, Widget? navigator) => const Example()));
+}
+
+class Example extends StatefulWidget {
+  const Example({Key? key}) : super(key: key);
+
+  @override
+  State<Example> createState() => _ExampleState();
+}
+
+class _ExampleState extends State<Example> {
+  final Runtime _runtime = Runtime();
+  final DynamicContent _data = DynamicContent();
+  late final WasmInstance _logic;
+
+  @override
+  void initState() {
+    super.initState();
+    RendererBinding.instance!.deferFirstFrame();
+    _runtime.update(const LibraryName(<String>['core', 'widgets']), createCoreWidgets());
+    _loadLogic();
+  }
+
+  late final WasmFunction _dataFetcher;
+
+  Future<void> _loadLogic() async {
+    final DateTime expiryDate = DateTime.now().subtract(const Duration(hours: 6));
```

Comment:
expireDate, right

Reply:
https://en.wikipedia.org/wiki/Expiration_date#:~:text=expiry%20date is what I was going for.

Reply:
ah, ok